tutorials/FusionExporter.md

```diff
@@ -122,7 +120,11 @@ Below is a basic overview of the design hierarchy rules that the exporter expect
 
 #### 1. Grounded Node
 
-You must ground one of your parts in the design. This tells Synthesis where to start branching off the rest of the nodes from. In the browser, you'll see there is a main root component. All other components under this root component will actually be used in the export. **NOTE**: Generally anything that is *underdefined* or *"disjointed"* from the rest of the design will be added under the main grounded object, so if objects that are supposed to be moving relative to what you define as grounded aren't, that is likely why.
+You must ground one of your parts in the design. To do this, you must pin a component.
```

Comment:
Factually "ground" has a specific meaning in Fusion and I think it's just not the right word for what we are requiring